### DISCUSSION TITLE
GeoJSON rendering is broken

### DISCUSSION THREAD

**Discussion by MiracleXYZ** [community/community#19258] [Other Feature Feedback, Questions, & Ideas]
The GeoJSON example in this [documentation page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#using-geojson) is now broken, only showing an error message of:

> Cannot read properties of undefined (reading 'type')

---

The previous error message is for Chrome. On Safari, that error message changes to

> undefined is not an object (evaluating 't.type')

**Comment by MiracleXYZ** [comment]
Just demonstrating this issue:

```geojson
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "id": 1,
      "properties": {
        "ID": 0
      },
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
              [-90,35],
              [-90,30],
              [-85,30],
              [-85,35],
              [-90,35]
          ]
        ]
      }
    }
  ]
}
```

**Reply by miha42-github** [reply]
I have exactly the same problem on Safari, but it appears to kinda render on Chrome but it is really zoomed in.  I am getting some additional error messages in Safari via the Javascript console.

<img width="1622" alt="Screenshot 2024-01-20 at 1 51 58 PM" src="https://github.com/community/community/assets/10818650/8377b23e-a037-4749-905e-ad7b10cef924">

**Reply by miha42-github** [reply]
While I cannot say if this is the problem it looks like there is a certificate matter to be tended to between GitHub and Azure maps.  Here's an updated screenshot.

<img width="1622" alt="Screenshot 2024-01-20 at 2 15 32 PM" src="https://github.com/community/community/assets/10818650/476c4c52-77a0-4a77-a1f8-4ea44d90c87f">

**Reply by miha42-github** [reply]
Finally, this kinda works in Chrome, but the initial rendering (see below):

<img width="1467" alt="Screenshot 2024-01-20 at 2 18 19 PM" src="https://github.com/community/community/assets/10818650/0ef846ba-89af-4aca-a15b-75f86db0bd15">

Isn't optimal, you have to zoom out (see below) to get to something that looks reasonable.  From what I can tell this is a bug in how markdown rendering is working on GitHub and should be tended to.

<img width="1467" alt="Screenshot 2024-01-20 at 2 19 22 PM" src="https://github.com/community/community/assets/10818650/b2d66845-9f26-4fce-b8fa-e91d372cb35b">

Additionally, while zoom out works (had to press it 12x to get to that view) zoom in doesn't.
